trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/get-fee-record-item-index-to-data-sort-value-map-helper.ts

```diff
@@ -0,0 +1,35 @@
+import orderBy from 'lodash.orderby';
+import { Currency } from '@ukef/dtfs2-common';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { FeeRecordItem } from '../../../api-response-types';
+
+export type GetFeeRecordItemIndexToDataSortValueMapItem = {
+  currency?: Currency | undefined;
+  amount?: number | undefined;
+  feeRecordIndex: number;
+};
```

Comment:
I'd really like to remove the term 'fee record' from this file. The main reason being the logic in this file really doesn't care that this is to do with fee records. We can also remove the reference to the word 'index' since this file only cares that we are giving it a key it should use for the dictionary and is not using it as an index.

Perhaps we can instead have:
```
export type GetKeyToCurrencyAndAmountSortValueMapItem = {
  currencyAndAmount?: CurrencyAndAmount | undefined,
  key: number;
};
```
Then our function could be named similarly, e.g. `getKeyToCurrencyAndAmountSortValueMap`.